File: JPAEventTracker/src/main/java/com/skilldistillery/jpaeventtracker/entities/Task.java
package com.skilldistillery.jpaeventtracker.entities;

import java.time.LocalDateTime;

import javax.persistence.Entity;

@Entity
public class Task {
	
	private int id;
	
	private String name;
	
	private LocalDateTime createdAt;
	
	private LocalDateTime updatedAt;
	
	private LocalDateTime dueDate;
	
	private Status status;

}

File: ngEvents/src/app/models/task.ts
export class Task {

  id: number;
  name: string;
  status: string;
  dueDate: Date;
  createdAt: Date;
  updatedAt: Date;

  constructor(id?: number, name?: string, status?: string, dueDate?: Date, createdAt?: Date, updatedAt?: Date){
    this.id = id;
    this.name = name;
    this.status = status;
    this.dueDate = dueDate;
    this.createdAt = createdAt;
    this.updatedAt = updatedAt;
  }
}
